ts/components/conversation/ImageGrid.tsx

```diff
@@ -1,263 +1,142 @@
 import React, { useContext } from 'react';
-import classNames from 'classnames';
 
 import {
   areAllAttachmentsVisual,
   AttachmentType,
   AttachmentTypeWithPath,
   getAlt,
-  getImageDimensionsInAttachment,
   getThumbnailUrl,
   isVideoAttachment,
 } from '../../types/Attachment';
 
 import { Image } from './Image';
 import { IsMessageVisibleContext } from './message/message-content/MessageContent';
+import styled from 'styled-components';
+import { THUMBNAIL_SIDE } from '../../types/attachments/VisualAttachment';
 
 type Props = {
   attachments: Array<AttachmentTypeWithPath>;
-  bottomOverlay?: boolean;
   onError: () => void;
   onClickAttachment?: (attachment: AttachmentTypeWithPath | AttachmentType) => void;
 };
+
+const StyledImageGrid = styled.div`
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  gap: var(--margins-sm);
```

Comment:
Nice use of flex gap! I need to start using this 😄

Reply:
:+1: